clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -234,11 +235,20 @@ public void testAutocommit_ResendAutocommitAfterException() {
 
     @Test
     public void testAutocommit_EnsureOnlyOneInflightRequest() {
+        TopicPartition t1p = new TopicPartition("topic1", 0);
+        subscriptionState.assignFromUser(singleton(t1p));
+        //subscriptionState.seekUnvalidated(t1p, new SubscriptionState.FetchPosition(100L));
```

Comment:
nit: maybe remove the commented line  if we do not need it

Reply:
oh yes... 🤦